maintainers/maintainer-list.nix

```diff
@@ -7235,6 +7235,17 @@
     githubId = 1267527;
     name = "Daniel Firth";
   };
+  lockejan = {
```

Comment:
This should be a separate commit: `maintainers: add lockejan`

Reply:
Makes sense. I someone guessed it is preferred to put it into one commit. At least I found a PR which hasn't had a distinct commit for that but I'm all for atomic commits. I probably just found a bad example.